sys/drv/bcm2835_gpio.c

```diff
@@ -21,9 +26,18 @@ static void delay(int64_t count) {
 }
 
 #define NGPIO_PIN 54
+#define FDT_PINS_INVAL (-1)
+
+#define BCM2835_PULL_NONE 0
+#define BCM2835_PULL_DOWN 1
+#define BCM2835_PULL_UP 2
```

Comment:
Where are you using that?

Reply:
Nowhere. Those were supposed to be used within DTS, but I ditched that idea. Removed.